uisdk/src/main/java/tech/dojo/pay/uisdk/DojoSDKDropInUI.kt

```diff
@@ -79,4 +81,13 @@ object DojoSDKDropInUI {
         if (requestCode != REQUEST_CODE_DROP_IN_UI) return null
         return DojoPaymentFlowHandlerResultContract().parseResult(resultCode, intent)
     }
+
+    /**
+     * Set Locale to be used in all the flows, will force ui to use the passed locale instead of the device locale.
+     * If language is not recognized or supported ui will fallback to english.
+     * If no language is set we will use the device locale with the supported languages
+     */
+    fun setSdkLocale(language: String) {
```

Comment:
i think we need to update the docs with that one

Reply:
I'm agree. I was thinking to wait ios for that. Now they are still back on version 1.4.x because on some merge pending to do, this week they should merge and align. We have a ticket to update the documentation on our sprint.